package.json

```diff
@@ -1,14 +1,21 @@
 {
   "name": "@air/react-memoized-context",
+  "version": "1.0.0",
```

Comment:
```suggestion
```
I don't think you need to version it - and I'm curious to see if it will handle it for us. Also, I think we should start at `0.0.1` just cause it's the first and once we know it's solid, then we can major bump